.circleci/config.yml

```diff
@@ -202,6 +229,7 @@ workflows:
     jobs:
       - go-checks
       - terraform-checks
+      - compile_weekly
```

Comment:
The job name `compile_weekly` and Make command `test-compile-weekly` confused me quite a bit to think this job is compiled weekly, instead of this compiles weekly tests. A small nit of a name suggestion: `weekly-job-checks`

Reply:
Thanks for the suggestion! That's a really good point - the name can be easily read in another way that is unintended.

🤔 the "weekly" word is kind of tricky. What are your thoughts on `compile-weekly-tests`? I'm afraid `weekly-job-checks` could be interpreted as weekly job-checks i.e. a job check that occurs weekly.

Other brainstormed ideas: `check-weekly-jobs`, `check-weekly-tests`

Reply:
ha yeah I can see how the `weekly-job-checks` suggestion runs into the same issue now. `compile-weekly-tests` is more clear than our first two ideas!

Reply:
Awesome, I'll go with that then. Thanks for chatting about this!